src/main/webapp/i18n/en/messages.json

```
@@ -0,0 +1,20 @@
+{
+    "artemisApp": {
+        "messages": {
+            "label": "Messages",
+            "conversation": {
+                "header": "Conversations",
+                "created": "Conversation created",
+                "show": "Show conversations",
+                "hide": "Hide conversations",
+                "notFound": "No conversations found",
+                "searchForUsers": "Search users within course"
+            },
+            "conversationParticipant": {
+                "created": "Conversation Participant created",
+                "notFound": "No conversationParticipant found"
```

```suggestion
                "notFound": "No Conversation Participant found"
```
Consistency and avoiding exposing internal implementation details.

---

removed as mentioned above